cloud-function/src/env.ts

```diff
@@ -46,7 +46,11 @@ export function getOriginFromRequest(req: Request): Origin {
     !req.path.endsWith("/runner.html")
   ) {
     return Origin.liveSamples;
-  } else if (req.hostname.endsWith(ORIGIN_PLAY)) {
+  } else if (
+    req.hostname.endsWith(ORIGIN_PLAY) ||
+    // In case ORIGIN_PLAY is not a prefix of ORIGIN_LIVE_SAMPLES.
+    (req.hostname === ORIGIN_LIVE_SAMPLES && req.path.endsWith("/runner.html"))
```

Comment:
If this is specifically for xyz where we don't set `ORIGIN_PLAY`, can we check for it:

```suggestion
    ORIGIN_PLAY
      ? req.hostname.endsWith(ORIGIN_PLAY)
      : // In case ORIGIN_PLAY is not a prefix of ORIGIN_LIVE_SAMPLES.
        req.hostname === ORIGIN_LIVE_SAMPLES &&
        req.path.endsWith("/runner.html")
```

Reply:
ORIGIN_PLAY has a default (I think). But anyway. Let's not do this in this PR and just use the stage runner.